### PR TITLE
made sure that every elements in a card header and heading are centered

### PR DESCRIPTION
**Pull request by fraincs**
Issue: 

## Summary

#934 - A card header doesn't center it's content vertically 

## What I did

Set display flex to a heading and header and centered the items.

## How to test

Add a Switch in an Card heading and header

### CODE REVIEW

File: packages/components/src/card/src/Card.css [hunk 1]
```diff
@@ -98,25 +103,25 @@
...
+.o-ui-card-vertical>.o-ui-card-aside {
     grid-column-start: 1;
     grid-column-end: 3;
     padding: var(--o-ui-sp-5);
 }
 
-.o-ui-card-vertical > .o-ui-card-illustration + .o-ui-card-aside,
-.o-ui-card-vertical > .o-ui-card-image + .o-ui-card-aside {
+.o-ui-card-vertical>.o-ui-card-illustration+.o-ui-card-aside,
+.o-ui-card-vertical>.o-ui-card-image+.o-ui-card-aside {
```
Comment (alexasselin008) [inline packages/components/src/card/src/Card.css:124]: There is a lot of css files with spaces around > and +. Should you fix them too?

Comment (fraincs) [reply packages/components/src/card/src/Card.css:124]: space should be there actually easier to read

Comment (patricklafrance) [reply packages/components/src/card/src/Card.css]: @fraincs have a look also in https://github.com/gsoft-inc/sg-orbit/pull/940/files then, space have been removed around the +

**Comment by patricklafrance** [comment]
How can I test this if this now work? Was there something like a failing chromatic tests before the fix?

**Comment by patricklafrance** [comment]
Most of the time when fixing something it's a good idea to had a test before to ensure we do not have any regression in the future.

**Review by LaurentDario — APPROVED** [review]

**Pull request merged**